src/modules/fundingManager/rebasing/abstracts/ElasticReceiptTokenBase_v1.sol

```diff
@@ -550,15 +557,21 @@ abstract contract ElasticReceiptTokenBase_v1 is IRebasingERC20, ERC165 {
     ///      updates the bit-tokens conversion rate and the total token supply.
     function _rebase() private {
         uint supplyTarget = _supplyTarget();
+        uint activeBits = _activeBits();
 
         // Do not adjust supply if target is outside of valid supply range.
-        // Note to not revert as this would make transfer's impossible.
-        if (supplyTarget == 0 || supplyTarget > MAX_SUPPLY) {
+        // Note: to not revert as this would make transfer's impossible.
+        if ((supplyTarget == 0 && activeBits == 0) || supplyTarget > MAX_SUPPLY)
```

Comment:
Added `activeBits == 0` as an additional condition where this should `return` (i.e. not do anything), which is only the case if no deposits happened. If the funds were used by the workflow, `activeBits` is greater than zero. Was necessary to allow the `supplyTarget` to be zero.